cmd/tailscale/cli/set_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cli
+
+import (
+	"net/netip"
+	"reflect"
+	"testing"
+
+	"tailscale.com/ipn"
+	"tailscale.com/net/tsaddr"
+)
+
+func ptrTo[T any](v T) *T { return &v }
+
+func TestCalcAdvertiseRoutesForSet(t *testing.T) {
+	pfx := netip.MustParsePrefix
+	tests := []struct {
+		name     string
+		exit     *bool
+		routes   *string
+		existing []netip.Prefix
```

Comment:
"had" or "was" is a bit shorter? don't care, though

Reply:
Done